packages/devtools/lib/src/main.dart

```diff
@@ -72,10 +72,18 @@ class PerfToolFramework extends Framework {
     await addScreens();
     screensReady.complete();
 
+    final settingsScreen = getScreen('settings');
```

Comment:
nit. Make 'settings' a top level constant defined in the settings screen file and use it everywhere the word 'settings' is used. That will prevent a bug if someone changes the title of the screen to something slightly different.

Reply:
ok, will do when refactoring the SettingsScreen to have SettingsController